src/test/codewhisperer/service/keyStrokeHandler.test.ts

```diff
@@ -250,15 +235,15 @@ describe('keyStrokeHandler', function () {
             await keyStrokeHandler.invokeAutomatedTrigger('Enter', mockEditor, mockClient, config)
             assert.ok(getRecommendationsStub.calledOnce || oldGetRecommendationsStub.calledOnce)
         })
+    })
 
-        it('should reset invocationContext.keyStrokeCount to 0', async function () {
-            const mockEditor = createMockTextEditor()
-            KeyStrokeHandler.instance.keyStrokeCount = 10
-            sinon
-                .stub(RecommendationHandler.instance, 'getServerResponse')
-                .resolves([{ content: 'import math' }, { content: 'def two_sum(nums, target):' }])
-            await KeyStrokeHandler.instance.invokeAutomatedTrigger('Enter', mockEditor, mockClient, config)
-            assert.strictEqual(KeyStrokeHandler.instance.keyStrokeCount, 0)
+    describe('shouldTriggerIdleTime', function () {
+        it('should return false when inline is enabled and inline completion is in progress ', function () {
+            const keyStrokeHandler = new KeyStrokeHandler()
+            sinon.stub(InlineCompletionService.instance, 'isPaginationRunning').returns(true)
+            sinon.stub(InlineCompletion.instance, 'isPaginationRunning').returns(true)
+            const result = keyStrokeHandler.shouldTriggerIdleTime()
+            assert.strictEqual(result, false)
```

Comment:
`shouldTriggerIdleTime` could be changed to accept instances of `InlineCompletion` and `InlineCompletionService` as params to make things more explicit.

Reply:
We are going to remove **InlineCompletion** soon since most people are on 1.68+ and they should be only using **InlineCompletionService**